api/proto/config/storage.proto

```diff
@@ -126,6 +126,14 @@ message ContentTree {
   uint64 maxSizeBytes = 6; // used for capping resource consumption in EVE
   SignatureInfo siginfo = 7;
   string displayName = 8; // Optional friendly name echo'ed in info
+
+  // For container ContentTree, sha256 won't be available in config
```

Comment:
Might not be available; we can still have a CT specified with a sha256, right?

Reply:
Yes. Resolved.